main_logic/omni_realtime_client.py

```
@@ -836,6 +836,23 @@ async def _create_response_gemini(self, instructions: str) -> None:
         except Exception as e:
             logger.error(f"Error sending client content to Gemini: {e}")
 
+    async def stream_proactive(self, instruction: str) -> bool:
```

Ruff (0.15.1)

[warning] Unused method argument: `instruction`

(ARG002)